processor/src/main/java/org/mapstruct/ap/internal/model/source/ForgedMethod.java

```diff
@@ -57,12 +57,12 @@
      */
     public ForgedMethod(String name, Type sourceType, Type targetType, MapperConfiguration mapperConfiguration,
         ExecutableElement positionHintElement) {
-        String sourceParamName = Strings.decapitalize( sourceType.getName().replace( "[]", "" ) );
```

Comment:
That moved into `Strings.sanatizeIdentifierName`, which is called by `Strings.getSaveVariableName` now - so this is obsolte here.